docs/migrating-from-opencensus.md

```diff
@@ -0,0 +1,47 @@
+## Action Plan for Bootstraping from OpenCensus
+
+### Goals
+We need to bootstrap the OpenTelemetry Service using the existing OpenCensus Service codebase. We agreed to split the Service codebase into 2 parts: core and contrib. This bootstrapping is a good opportunity to do the splitting by only including in the OpenTelemetry Service core the minimum number of receivers and exporters and moving the rest of functionality to a contrib package (most vendor-specific code).
+
+The contrib package and vendor-specific receivers and exporters will continue to be available and there is no intent to retire it. The intent is to have a clear decoupling in the codebase that facilitates independent contribution of new components in the future, allows to easily create customized versions of a Service and makes it clear that core contributors will be responsible for maintenance of the core while vendor-specific components will be maintained by corresponding vendors (note: this does not exclude dual participation at all - some developers will likely work for vendors and will also be core maintainers).
+
+# Migration Tasks
+
+This is the action plan that also shows the progress. Tick the boxes after the task is complete.
+
+[ ] Copy all commits from https://github.com/census-instrumentation/opencensus-service to https://github.com/open-telemetry/opentelemetry-service
+Make sure commit history is preserved.
```

Comment:
in .net we intentionally went with shallow commit to make sure there is no shady area around people names on commits when CNCF CLA was not signed by them.

I, however, have no problems with preserving history. It may be more important in service repo.

Reply:
My understanding is that all commits in OpenCensus already require CLA to be signed before they get merged, so hopefully this is not an issue. I think it is important to preserve the history, it helps new contributors to understand why a particular piece of code was written.